Documentation/project-docs/linux-performance-tracing.md

```diff
@@ -3,13 +3,44 @@
 
 When a performance problem is encountered on Linux, these instructions can be used to gather detailed information about what was happening on the machine at the time of the performance problem.
 
-# Required Tools #
+# EventPipe and dotnet-trace (.NET Core 3.0 Preview 5 or later)
```

Comment:
I think we'll need a little introductory material describing the different capabilities of dotnet-trace/EventPipe vs. perfcollect/lttng so that people can pick the right one before reading the technology specific sections. Important differences that come to mind:

1) perfcollect invokes perf, which collects callstacks for native code. Dotnet-trace/EventPipe only has managed callstacks.
2) perfcollect has machine-wide scope so you can start it prior to running a process and potentially capture multiple processes. dotnet-trace is runtime-wide scope and you start it after a process is running.
3) dotnet-trace is OS agnostic, perfcollect/lttng/perf is Linux-only.
4) dotnet-trace only being available starting in the 3.0 release

For developers that are looking in this GitHub repo because they are trying to diagnose the behavior of native code in the runtime, the native callstacks and ability to get process startup probably make perfcollect preferable. For many other customers dotnet-trace is likely preferable. Given where these docs are I would bias towards runtime devs and put perfcollect at the top - in most other contexts I would order dotnet-trace first as you did here.